scripts/near-shell-check.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file is a stop-gap for upgrading projects when near-shell improves/modifies logic
+ */
+
+const fs = require('fs');
+const util = require('util');
+
+(async () => {
+    const oldDevDeployFile = 'neardev/dev-account';
+    const newDevDeployFile = 'neardev/dev-account.env';
+
+    if (fs.existsSync(newDevDeployFile)) {
+        // user already has the latest near-shell
+        return;
+    }
+
+    if (fs.existsSync(oldDevDeployFile)) {
+        // user has an outdated near-shell, create necessary file
+        const readFile = (filePath) => util.promisify(fs.readFile)(filePath, 'utf8');
```

Comment:
Note that there is API already using promises https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api

Reply:
I remember reading that these weren't in Node 10 when I did this, but now that we've bumped to 12 no reason not to.

Reply:
Welp, these docs are for 10 so I guess whatever I read was wrong

Reply:
k, pushed up a change. I do think it's important to have this migration.
A lot of end users are hobbyists and may have downloaded older versions of `near-shell` I would like to have a stop-gap for this case.

Reply:
> A lot of end users are hobbyists and may have downloaded older versions of near-shell I would like to have a stop-gap for this case.

so next `yarn dev-deploy` for them would just create new account and everything would still work fine, these are throwaway accounts in any case